cross-media-measurement/src/main/proto/wfa/measurement/common/crypto/liquid_legions_v2_encryption_methods.proto

```diff
@@ -212,14 +212,18 @@ message CompleteExecutionPhaseTwoAtAggregatorRequest {
   // The elliptical curve to work on for SKA generation.
   int64 curve_id = 4;
   // Maximum frequency to measure. At least 2.
-  // (maximum_frequency-1) will is the row dimension of the 2-D SKA matrix.
+  // (maximum_frequency-1) will be the row dimension of the 2-D SKA matrix.
   int32 maximum_frequency = 5;
   // LiquidLegions parameters used for reach estimation.
   LiquidLegionsParameters liquid_legions_parameters = 6;
   // Parameters for computing the noise baseline of the global reach DP noise
   // registers added in the setup phase.
   // The baseline is subtracted before reach is estimated.
-  GlobalReachDpNoiseBaseline noise_baseline = 7;
+  GlobalReachDpNoiseBaseline reach_dp_noise_baseline = 7;
```

Comment:
I remember that when we first discussed the three flags-approach, we were thinking of using the number of non-empty registers from the end of Execution Phrase I (right after SKA, and before adding frequency noise). Doing so wouldn't require passing this reach_dp_noise_baseline for each estimation. Is there a reason we change to this?

Reply:
do you mean "Doing so wouldn't require passing this frequency_noise_parameters for reach estimation"

We need to filter out the histogram noise before estimating reach, since that noise increases linearly as the number of publishers. such we couldn't estimate reach right after SKA. The filtering happens in round 2.

Reply:
I understand your point, but I thought we would just record that number and the subtract the blind histogram noise in this stage. Anyway, either works so I guess it doesn't matter too much.